core/capabilities/targets/monitoring.go

```diff
@@ -0,0 +1,38 @@
+package targets
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+	"github.com/smartcontractkit/chainlink-common/pkg/metrics"
+
+	localMonitoring "github.com/smartcontractkit/chainlink/v2/core/monitoring"
+)
+
+type writeTargetMetricsLabeler struct {
```

Comment:
What's the goal of this metric? If we are going to emit failure metrics at the capability level, will this overlap with the capability level metrics you have PRs up for in the engine?

Reply:
Note that we are looking to add capability ID to at least one metric that tells us about step failures: https://github.com/smartcontractkit/chainlink/pull/15238/files#r1846939651